### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by tobiaserich**

### CODE REVIEW

File: client/src/components/DescriptionInput.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  padding-left: 10px;
+  padding-top: 5px;
+  padding-right: 10px;
+  font-size: 16px;
+  background-color: ${({ theme }) => theme.colors.active};
+`;
+
+export default function DescriptionInput() {
+  return <TextInput />;
+}
```
Comment (marwinburesch) [inline client/src/components/DescriptionInput.js]: You don't necessarily need a `React` functional component here. The styled component works just fine

File: client/src/components/Input.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  border: none;
+  border-radius: 1px;
+  font-size: 18px;
+  margin: auto;
+  margin-top: 10px;
+  padding-left: 5px;
+`;
+
+export default function Input({ name }) {
+  return <InputField name={name} />;
```
Comment (marwinburesch) [inline client/src/components/Input.js]: I don't see the use of the property `name` here, all you do is pass it along and then not use it :)
I think you could just go with the styled component here as well

File: client/src/components/AddEventCard.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import styled from "@emotion/styled";
+import PropTypes from "prop-types";
+
+const Card = styled.form`
+  width: 280px;
+  height: 400px;
+  background: ${({ theme }) => theme.colors.cardBackground};
```
Comment (marwinburesch) [inline client/src/components/AddEventCard.js]: `${theme => theme.colors.cardBackground}` would work just fine I'd say

File: client/src/components/PersonalInformation.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+const CustomHeader = styled.h3`
+  margin: 0;
+  font-weight: 500;
+`;
+
+export default function PersonalInformation({ name, birthday, gender, address }) {
+  return (
+    <InformationContainer>
+      <DetailContainer>
```
Comment (marwinburesch) [inline client/src/components/PersonalInformation.js]: I like that you reuse this component 🐜

File: client/src/components/PollInfo.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+`;
+
+const StatusImage = styled.img`
+  max-width: 25px;
+`;
+const pollStatus = {
+  blank: blank,
+  pending: pending,
+  finished: finished
+};
```
Comment (marwinburesch) [inline client/src/components/PollInfo.js]: nice

File: client/src/components/Input.js (hunk 2 above)
Comment (tobiaserich) [reply client/src/components/Input.js]: Thanks, i even don't know why i did this :D .

File: client/src/components/AddEventCard.js (hunk 3 above)
Comment (tobiaserich) [reply client/src/components/AddEventCard.js]: i tried it but theme is undefined. 
I have to destructure the props in {theme }

**Pull request merged**